tests/unit/docs/test_sharedexample.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
It would be nice if we can add a test in either ``test_client.py`` or ``test_service.py`` to ensure that the example get plummed into the documentation for a client.